core/src/forwarding_stage.rs

```diff
@@ -201,6 +201,7 @@ impl<F: ForwardAddressGetter> ForwardingStage<F> {
                 let Some(packet_data) = packet.data(..) else {
                     // should never occur since we've already checked the
                     // packet is not marked for discard.
+                    datapoint_error!("forward_stage_packet_discard_error", ("count", 1, i64));
```

Comment:
is there some other related metrics struct nearby that we could accumulate into first? getting a bunch of these could flood the metrics db

Reply:
It should never happen though. Honestly I'd prefer panicking than having an always 0 metric that gets reported every second

Reply:
you won't do it

Reply:
[389632a](https://github.com/anza-xyz/agave/pull/4991/commits/389632ae1b4a7b34782028ac368326e9d3a6e135)